lib/utilities.js

```diff
@@ -2,6 +2,9 @@
 
 const path = require('path');
 
+const UPPER_LETTERS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+const LOWER_LETTERS = 'abcdefghijklmnopqrstyvwxyz';
```

Comment:
I'd prefer to compare `charCodeAt` with `a` and `z` instead of finding char in strings `UPPER_LETTERS` and `LOWER_LETTERS`

Reply:
Done